plasmapy/analysis/sig_analysis.py

```diff
@@ -0,0 +1,75 @@
+"""_summary_
+"""
+
+import numpy as np
+import scipy.io as spio
+import scipy.integrate as sp
+from scipy import signal
```

Comment:
> Additonally, I can't change the name to signal.py because that causes a circular import issue. Numpy already has a module called signal.py

That's not technically a circular import issue. You can, since it's sort of necessary here, do

```suggestion
import scipy.signal
```

And then you can refer to this later on as
```python
    b, a = scipy.signal.butter(order, normal_cutoff, btype='highpass', analog=False)
```